Add note about install python on alpine (#6677)
astral-sh/uv: docs/guides/integration/docker.md

RUN uv sync --frozen
```

!!! note

>     We currently don't support installing Python on musl-based distributions. If you are using an

Reviewer: ```suggestion
    uv does not yet support installing Python on musl-based distributions. For example, if you are using an
```

>     alpine base image that doesn't have Python installed, you need to add it manually:

Reviewer: ```suggestion
    Alpine Linux base image that doesn't have Python installed, you need to add it with
    the system package manager:
```

    in your repository to prevent it from being included in image builds. The project virtual
    environment is dependent on your local platform and should be created from scratch in the image.

> !!! note

Reviewer: Sorry I should have noticed this in the first run. This is kind of a weird spot? Can we put this in a new section at the bottom or something? It feels quite prominent here and we're not talking about installing Python at all.
